samples/get_sample_data.bash

```diff
@@ -0,0 +1,18 @@
+#! /usr/bin/env bash
+
+set -e
+
+# save a few directory names
+CWD=$PWD
+SAMPLES="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
```

Comment:
this is the full path of this file. (e.g. `/home/VIC/samples/get_sample_data.bash`)